src/GraphQl/Type/SchemaBuilder.php

```diff
@@ -156,19 +174,26 @@ private function getNodeQueryField(): array
     /**
      * Gets the query fields of the schema.
      */
-    private function getQueryFields(string $resourceClass, ResourceMetadata $resourceMetadata): array
+    private function getQueryFields(string $resourceClass, ResourceMetadata $resourceMetadata, string $operationName = 'query', $item, $collection): array
```

Comment:
```suggestion
    private function getQueryFields(string $resourceClass, ResourceMetadata $resourceMetadata, string $operationName, $itemConfiguration, $collectionConfiguration): array
```
I think you should add PHPDoc for the `$itemConfiguration` and `$collectionConfiguration` parameters like this:
```php
/**
 * @param array|bool $itemConfiguration false if not configured
 * @param array|bool $collectionConfiguration false if not configured
 */
```